fastlane/lib/fastlane/fast_file.rb

```diff
@@ -1,3 +1,26 @@
+require "rubygems"
+
+class String
+  # Gem::Version.new barfs on things like "v0.1.0", which is the style
+  # generated by the rake release task. Just strip off any initial v
+  # to generate a Gem::Version from a tag.
+  def fastlane_version_number_from_tag
+    sub(/^v/, "")
+  end
+end
+
+module Gem
+  class Version
+    def self.from_tag(tag)
```

Comment:
Sorry, missed that on the first run, we'll also need to prefix this method with `fastlane_`

Reply:
That makes for an ugly API. For example, what does `fastlane_from_tag` mean? Am I--creating a `fastlane` from a tag? But it's just as well not to put these in these classes, so I have a better idea. Commit incoming.